### PR TITLE
Update UserGuide on meeting

### PR DESCRIPTION
**Pull request by Pluiexo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -169,6 +169,23 @@ Examples:
...
   ![result for 'sort p/'](images/sortByPhoneNumberResult.png)
 
+### Adding a meeting to a person: `meeting`
+
+Add a meeting to a person based on specified description and date.
+
+Format: `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`
+
+* Adds a meeting to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3,…​ and tally within range index of the displayed list.
+* At both of the fields must be provided and valid.
```
Comment (iynixil) [inline docs/UserGuide.md]: nit: Should this just be `Both of the fields` without the "at"? Perhaps the latter part can also be edited: `and be valid values.`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -169,6 +169,23 @@ Examples:
...
+* A valid `DESCRIPTION` of the meeting can only contain latin alphanumeric characters.
+* A valid `DATETIME` of the meeting can only contain valid date and 24 hour time values with a single space to separate the date and time in the format of `dd/MM/yyyy HH:mm`.
+* Duplicates meetings with the same `DESCRIPTION` and `DATETIME` in the same person is not allowed.
```
Comment (iynixil) [inline docs/UserGuide.md]: nit: `Duplicate` without the "s"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -169,6 +169,23 @@ Examples:
...
+
+Examples:
+* `meeting 1 d/ Meet for finals preperation s/ 12/04/2024 18:00` adds a meeting to the first person with the description of `Meet for finals preperation` and the at the date and time of `12/04/2024 18:00`
```
Comment (iynixil) [inline docs/UserGuide.md]: nit: Should the latter part be `and the date and time of __` without the "at the"?
Also optional and super minor to fix: `preparation` typo (if this is fixed, maybe picture also need to change)

File: docs/UserGuide.md [hunk 4]
```diff
@@ -169,6 +169,23 @@ Examples:
...
+
+* Adds a meeting to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3,…​ and tally within range index of the displayed list.
+* At both of the fields must be provided and valid.
+* A valid `DESCRIPTION` of the meeting can only contain latin alphanumeric characters.
+* A valid `DATETIME` of the meeting can only contain valid date and 24 hour time values with a single space to separate the date and time in the format of `dd/MM/yyyy HH:mm`.
+* Duplicates meetings with the same `DESCRIPTION` and `DATETIME` in the same person is not allowed.
+
+Examples:
+* `meeting 1 d/ Meet for finals preperation s/ 12/04/2024 18:00` adds a meeting to the first person with the description of `Meet for finals preperation` and the at the date and time of `12/04/2024 18:00` 
+* `meeting 2 d/ Meet for practical exam s/ 20/04/2024 15:00` adds a meeting to the second person with the description of `Meet for practical exam` and at the date and time of `20/04/2024 15:00`
```
Comment (iynixil) [inline docs/UserGuide.md]: here too, same as above

File: docs/UserGuide.md [hunk 5]
```diff
@@ -251,5 +268,6 @@ Action | Format, Examples
 **Filter** | `filter [m/MODULE] [f/FACULTY] [t/TAG]…`<br> e.g., `filter m/CS2100 t/friends`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **Sort** | `sort [ATTRIBUTE]`<br> e.g., `sort p/`
+**AddMeeting** | `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`<br> e.g., `meeting 1 d/ Meet for finals preperation s/ 12/04/2024 18:00`
```
Comment (iynixil) [inline docs/UserGuide.md]: nit: optional and minor, `Add Meeting` with space between

**Review by iynixil — COMMENTED** [review]
looks good! minor nits and typos to fix

**Comment by Pluiexo** [comment]
> looks good! minor nits and typos to fix

Has been amended!

File: docs/UserGuide.md (hunk 5 above)
Comment (Pluiexo) [reply docs/UserGuide.md]: Now that i think about it, just ```Meeting``` would be better

**Review by iynixil — APPROVED** [review]

**Pull request merged**